examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -2577,6 +2657,10 @@ rtDefineProperty(pxScene2d, cors);
 rtDefineMethod(pxScene2d, addServiceProvider);
 rtDefineMethod(pxScene2d, removeServiceProvider);
 
+#ifdef PXSCENE_SUPPORT_STORAGE
+rtDefineProperty(pxScene2d, storage);
```

Comment:
Let's expose this property always but have it return RT_FAIL if storage is compiled off.